package.json

```diff
@@ -32,28 +33,32 @@
   "license": "MIT",
   "scripts": {
     "build:node": "esbuild src/node.ts --outfile=lib/node.cjs --bundle --sourcemap --platform=node --external:cosmokit --external:supports-color --target=es2019",
-    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:supports-color --target=esnext",
-    "build": "yarn build:node && yarn build:browser && yarn tsc -b",
+    "build:browser": "esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:util --target=esnext",
+    "build:deno": "esbuild src/deno.ts --outfile=lib/deno.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --target=esnext",
+    "build": "yarn build:node && yarn build:browser && yarn build:deno && yarn tsc -b",
     "test": "mocha -r esbuild-register tests/*.spec.ts",
     "test:text": "shx rm -rf coverage && c8 -r text yarn test",
     "test:json": "shx rm -rf coverage && c8 -r json yarn test",
-    "test:html": "shx rm -rf coverage && c8 -r html yarn test"
+    "test:html": "shx rm -rf coverage && c8 -r html yarn test",
+    "dep": "yakumo upgrade"
```

Comment:
We do not introduce yakumo because this is not a monorepo.

Reply:
> We do not introduce yakumo because this is not a monorepo.

Now removed